WordPress/gutenberg
# Revert "Build Tooling: Update development tools to work with PostCSS 8 (#27821)"

Something in b90e38c4e3dbcb716a3ea09e23ef9d8df76a9a50 has broken file watching in `npm run dev`. This reverts https://github.com/WordPress/gutenberg/pull/27821 so that developers can continue use `npm run dev` while the issue is investigated. 

To test:

1. `npm install`
2. `npm run dev`
3. Make a change to a JavaScript file.
4. The change should be visible in the generated `build/*.js` file.

Or, alternatively:

1. `npm install`
2. `npx wp-scripts start`
3. The process should not exit.
